config/runtime.exs

```diff
@@ -146,6 +146,12 @@ if System.get_env("RELEASE") == "central_backend" or config_env() == :dev do
   config :game_backend, :currencies_config, Jason.decode!(currency_config_json)
 end
 
+{:ok, daily_rewards_json} =
```

Comment:
Please try running a `arena` release as this might break it

Reply:
Done. Nice catch!